src/index.html

```diff
@@ -15,8 +15,29 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/"class="logo__img"><img src="./images/logo.png"alt="Moyo Logo"/></a>
```

Comment:
The `alt` attribute for the `<img>` tag should describe the image content. The current `alt` attribute is missing a proper description.